Swinject/Container.swift

```diff
@@ -100,7 +100,7 @@ public final class Container {
     ///
     /// - parameter loader: the loader to load properties into the container
     ///
-    public func applyPropertyLoader<T: PropertyLoaderType>(loader: T) throws {
+    public func applyPropertyLoader(loader: PropertyLoaderType) throws {
```

Comment:
Thank you for explaining the part. I got better understanding about the Swift generics.